lib/fastlane/plugin/wpmreleasetoolkit/actions/android/android_generate_apk_from_aab.rb

```diff
@@ -0,0 +1,132 @@
+module Fastlane
+  module Actions
+    class AndroidGenerateApkFromAabAction < Action
+      def self.generate_command(aab_file_path, apk_output_file_path, keystore_path, keystore_password, keystore_key_alias, signing_key_password)
+        command = "bundletool build-apks --mode universal --bundle #{aab_file_path} --output-format DIRECTORY --output #{apk_output_file_path} "
+        code_sign_arguments = "--ks #{keystore_path} --ks-pass #{keystore_password} --ks-key-alias #{keystore_key_alias} --key-pass #{signing_key_password} "
+        move_and_cleanup_command = "&& mv #{apk_output_file_path}/universal.apk #{apk_output_file_path}_tmp && rm -rf #{apk_output_file_path} && mv #{apk_output_file_path}_tmp #{apk_output_file_path}"
+
+        # Attempt to code sign the APK if a keystore_path parameter is specified
+        command += code_sign_arguments unless keystore_path.nil?
+
+        # Move and rename the universal.apk file to the specified output path and cleanup the directory created by bundletool
+        command += move_and_cleanup_command
```

Comment:
I've tried several approaches to deal with how `bundletool` handles the `--output` option and how it always defaults to the `universal.apk` filename, but I ran into all sorts of limitations and unnecessary complexity. Appending the move and cleanup commands as I've done here is the least complicated approach, especially with how I've structured the tests.

Reply:
I've solved this [using a slightly different but similar way that I suggested in the above comment](https://github.com/wordpress-mobile/release-toolkit/blob/1d566d1e235dad84f79783a82e70378e6cbb865a/lib/fastlane/plugin/wpmreleasetoolkit/actions/android/android_generate_apk_from_aab.rb#L15-L33). This should fix the issues the previous code was having with path containing spaces + security issues about risk of parameter injection + of removing entire user directory. lmkwyt.